Description: setup don't obey the settings of any englobing .pc 
 .
 This is mainly intended to get the setup.test working even if the
 debian package contains a .pc directory. Without this patch, the
 debian packaging stuff will get the testsuite using debian/patches
 instead of patches (because it's the way it goes in our .pc). The
 test breaks with that setting.
 .
 The patch changes the setup command to not take the settings of any
 .pc directory found, and reset QUILT_PC QUILT_PATCHES and QUILT_SERIES
 to their default values.
Bug-Debian: http://bugs.debian.org/573689
Forwarded: 2014-01-18

---
 quilt/setup.in |    5 +++++
 1 file changed, 5 insertions(+)

Index: b/quilt/setup.in
===================================================================
--- a/quilt/setup.in
+++ b/quilt/setup.in
@@ -17,8 +17,13 @@
 	. $QUILT_DIR/scripts/patchfns
 	if [ -n "$SUBDIR" ]
 	then
+		# Damn, found an enclosing quilt directory; don't follow its settings
 		cd $SUBDIR
 		unset SUBDIR
+		unset QUILT_PC QUILT_PATCHES QUILT_SERIES
+		: ${QUILT_PC:=.pc}
+		: ${QUILT_PATCHES:=patches}
+		: ${QUILT_SERIES:=series}
 	fi
 fi
 
